meta-oasis-vm/recipes-kernel/linux/linux-yocto-tiny_6.12.bb

```diff
@@ -17,8 +17,8 @@ DEPENDS += "openssl-native util-linux-native"
 KMETA = "kernel-meta"
 KCONF_BSP_AUDIT_LEVEL = "2"
 
-SRCREV_machine ?= "5b22e7aef539a991b75fa45101cbc02051327cb6"
-SRCREV_meta ?= "a50ff542f8d79235030741f9e850d34768907d2d"
+SRCREV_machine ?= "fee8195f8412ff8f1bfc50ba86cdb8f20de22750"
+SRCREV_meta ?= "f2f3b6cbd91743920e2cb55998c53326855b2e9c"
```

Comment:
Just curious. Do you know why the .kver was updated to v6.15 in this commit? (https://git.yoctoproject.org/yocto-kernel-cache/commit/?id=f2f3b6cbd91743920e2cb55998c53326855b2e9c)

Reply:
Good catch, no idea, looks like a mistake to me. Don't know if this has any effect.